types.d.ts

```diff
@@ -36,6 +36,6 @@ declare module 'geojson2h3' {
      */
     export function h3SetToFeatureCollection(
         hexagons: string[],
-        getProperties: (h3Index: string) => GeoJsonProperties
+        getProperties?: (h3Index: string) => GeoJsonProperties
```

Comment:
Technically I think this could be `(h3Index: string) => GeoJsonProperties | undefined`